From 910c0ec78b2e26757bfed1a45f496141855cff45 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Fri, 7 Jun 2024 15:16:06 +0800
Subject: [PATCH] Add miss <string>

---
 include/create/packet.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/create/packet.h b/include/create/packet.h
index d1e7928..60ff0f2 100644
--- a/include/create/packet.h
+++ b/include/create/packet.h
@@ -32,6 +32,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #define CREATE_PACKET_H
 
 #include <mutex>
+#include <string>
 
 namespace create {
   class Packet {
-- 
2.41.0

